app/Http/Livewire/LikeButton.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Livewire;
+
+use Livewire\Component;
+
+class LikeButton extends Component
+{
+    public $message;
+    public $likes;
+    public $icon;
+    private $isLike; // いいねしているか TODO いいね機能実装時に削除
```

Comment:
(should)
初期化タイミングがないので。

```suggestion
    private $isLike = false; // いいねしているか TODO いいね機能実装時に削除
```

Reply:
確かに真偽値として値を確定させておくことは重要ですね。そうするべきでした。#5 でその行は削除されるので、この PR では変更を加えないでマージしたいです。

(まずもって livewire を使う場合、状態保存にプロパティを使うことができないようです)

https://github.com/sya-ri/t5r/blob/2f0b095e2c8fef957c6155814a976029f1d53b7e/app/Http/Livewire/LikeButton.php#L11-L32

Reply:
了解！